src/crate/client/sqlalchemy/dialect.py

```diff
@@ -158,7 +159,11 @@ def process(value):
 class CrateDialect(default.DefaultDialect):
     name = 'crate'
     driver = 'crate-python'
-    statement_compiler = CrateCompiler
+    if SA_VERSION >= SA_2_0:
+        from .sa20 import CrateCompilerSA20
+        statement_compiler = CrateCompilerSA20
+    else:
+        statement_compiler = CrateCompiler
```

Comment:
I will apply this kind of modularization to the head branch first, to split specializations for SA14 into a separate `sa14.py` module, respectively a `CrateCompilerSA14` class, beforehand. After that, this patch will fit in more naturally.

Reply:
> I will apply this kind of modularization to the head branch first.

See GH-498.